usr/tgtd.c

```diff
@@ -925,7 +979,19 @@ static int target_delete(const _ha_request *reqp,
 			"tgt cmd too long");
 		return HA_CALLBACK_CONTINUE;
 	}
+retry_loop:
 	rc = exec(cmd);
+	if (rc == TGTADM_LUN_ACTIVE ||
+		rc == TGTADM_TARGET_ACTIVE ||
+		rc == TGTADM_DRIVER_ACTIVE ||
+		rc == TGTADM_UNSUPPORTED_OPERATION) {
+		fprintf(stderr, "Retrying for errno: %d\n", rc);
+		sleep(delay);
```

Comment:
Reduce the delay to 5 secs and correspondingly set the retry value. 10 secs seems a larger wait time for IOs to complete.

Reply:
Done